geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/MessageDispatcher.java

```diff
@@ -83,7 +84,7 @@ public class MessageDispatcher extends LoggingThread {
   /**
    * Default value in milliseconds for waiting for re-authentication
    */
-  private static final long DEFAULT_RE_AUTHENTICATE_WAIT_TIME = 60 * 1000;
+  private static final long DEFAULT_RE_AUTHENTICATE_WAIT_TIME = Duration.of(60, SECONDS).toMillis();
```

Comment:
Going to split some hairs here, but why make a humans do math in their heads?
```java
Duration.of(1, MINUTES).toMillis()
```

Reply:
Or: `Duration.ofMinutes(1).toMillis()`. Or declare the constant as a `Duration` and convert it only when passing it to something that needs millis.